### DISCUSSION TITLE
Order of install

### DISCUSSION THREAD

**Discussion by sreejesh123** [strimzi/strimzi-kafka-operator#5273] [General]
Hi I am new to strmzi kafka  or any kafka :-).   the order of install of strmzi is it operator,   kafka-persistent (kafka+ZK), User operator , Topic operator?    I did find the helm install but there is no much on how to get the order post operator install. any help or RTFM  me to the correct document will be really appreciated.

**Comment by scholzj** [comment]
You always have to first install the Cluster operator. That is in your case what the Helm Chart should do. Once the operator is running, you just create the custom resources describing the operands you need. For example, to deploy Kafka cluster, you create the `Kafka` custom resource which deploys first ZooKeeper, then Kafka and at the end the user and topic operators (you do not have to deploy them separately, they are part of the Kafka cluster). If you deploy something like Connect, which will use the Kafka cluster, you usually deploy it only after the Kafka cluster is ready. But even if you do it earlier it should eventually connect and get ready as well.

Give a try to the quickstart docs - they should take you through this.
